xml/crm.dtd

```diff
@@ -65,7 +65,7 @@ Additionally, num_peers and have_quorum are used during the election process to
  * ccm_transition stores the membership instance from the ConsensusClusterMembership layer.
  * cib_feature_revision is the feature set that this configuration requires
 -->
-<!ELEMENT configuration (crm_config, nodes, resources, constraints)>
+<!ELEMENT configuration (crm_config, nodes, resources, constraints, alerts)>
```

Comment:
A mystery to observe I have still on my list. Till I had done this alerts wanted to live on top
level instead inside configuration as I remember. I'll observe that again...